app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt

```diff
@@ -107,6 +113,37 @@ class TopicInfoFragmentTest {
   }
 
   // TODO(#914): Add more data to topic description so that these test can pass on XHDPI and XXHDPI and XXXHDPI devices.
+  @Test
+  fun testTopicInfoFragment_loadFragment_MoreThanFiveLines_seeMoreIsVisible() {
```

Comment:
Rename this to: `testTopicInfoFragment_loadFragment_moreThanFiveLines_seeMoreIsVisible`

Reply:
Done